sdk/core/azure-core/tests/async_tests/test_rest_stream_responses_async.py

```diff
@@ -138,9 +138,7 @@ async def test_cannot_read_after_response_closed(port, client):
 @pytest.mark.asyncio
 async def test_decompress_plain_no_header(client):
     # thanks to Xiang Yan for this test!
-    account_name = "coretests"
-    url = "https://{}.blob.core.windows.net/tests/test.txt".format(account_name)
-    request = HttpRequest("GET", url)
+    request = HttpRequest("GET", "/streams/string")
```

Comment:
We already have tests using test server. Don't we?

Reply:
For `azure-core`, these test the newer `rest.HttpRequest`, so a slightly different code path. Will keep these in. 

However, for `core-http`, there are tests in `test_streaming(_async).py` test the same thing as the tests I adjusted here, so I will remove them.

Also, turns out the tests marked `live_test_only` haven't even been running all this time since we didn't configure the `AZURE_TEST_RUN_LIVE` env var in `tests.yml`. Since all of these are using the now inaccessible storage account, I think it's time to just remove them as well.